app/models/teacher_project_views.rb

```diff
@@ -0,0 +1,9 @@
+class TeacherProjectViews < ActiveRecord::Base
+  self.table_name = :teacher_project_views
+
+  attr_accessible :viewed_projects_id, :teacher_id, :created_at, :updated_at
```

Comment:
I'd think that `viewed_projects_id` should be `viewed_project_id` which probably means changing the migration.